hack/edit-spi-config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
```

Comment:
can you add additional information about what this script is made for?

Reply:
👍🏼